### PR TITLE
[bundle-size] Verify whether a pull request is approved by a reviewer using the `approving_teams` field

### PR DESCRIPTION
**Pull request by danielrozenberg**
#617

For now this only logs the results, but does not assign or resolve the check - checks are resolved using the legacy logic until we're satisfied with the logs

### CODE REVIEW

File: bundle-size/webhooks.js [hunk 1]
```diff
@@ -80,52 +80,74 @@ exports.installGitHubWebhooks = (app, db, githubUtils) => {
...
+    }
 
-      let approvalMessagePrefix;
-      if (check.delta === null) {
-        context.log(
-          'Pull requests can no longer be preemptively approved for ' +
-            'bundle-size changes'
-        );
+    context.log(
+      `Pull request ${pullRequestId} approved by a bundle-size keeper`
```
Comment (rcebulko) [inline bundle-size/webhooks.js]: Where did "keeper" come from? :soccer:

File: bundle-size/webhooks.js [hunk 2]
```diff
@@ -80,52 +80,74 @@ exports.installGitHubWebhooks = (app, db, githubUtils) => {
...
+    await context.github.checks.update({
+      owner: check.owner,
+      repo: check.repo,
+      check_run_id: check.check_run_id,
+      conclusion: 'success',
+      completed_at: new Date().toISOString(),
+      output: {
+        title: `${approvalMessagePrefix} | approved by @${approver}`,
+        summary:
+          'The bundle size (brotli compressed size of `v0.js`) of this pull ' +
```
Comment (rcebulko) [inline bundle-size/webhooks.js]: Doesn't this apply to other binaries (ie. not just `v0.js`)?

File: bundle-size/webhooks.js (hunk 1 above)
Comment (danielrozenberg) [reply bundle-size/webhooks.js]: Old text that will be changed when we move to the new model

File: bundle-size/webhooks.js (hunk 2 above)
Comment (danielrozenberg) [reply bundle-size/webhooks.js]: Same

**Pull request merged**